redis.conf

```diff
@@ -498,12 +498,24 @@ slave-priority 100
 #
 # Please check http://redis.io/topics/persistence for more information.
 
-appendonly no
+appendonly yes
 
 # The name of the append only file (default: "appendonly.aof")
 
 appendfilename "appendonly.aof"
 
+# AOF will be written to NVML's PMEMlog pool. NOTICE: when aof-use-nvml is
+# enabled appendfsync option will be ignored.
```

Comment:
I would add more info here:
1) appendfsync is ignored - all the writes are flushed immediately, so in practice, it behaves like it's forced to "always"
2) More info about differences between standard AOF file and PMEMlog pool. I.e.:
- explain that there are two AOF files (memory mapped) at all the time, and we only switch between them when performing AOF rewrite.
- the NVML-based AOF size is fixed (it does'n grow up when necessary), so the log size must be large enough
- using NVML-based AOF doesn't make a big sense with a standard filesystem, as msync() is the performance killer.